### ISSUE TITLE
Question concerning temporal fusion

### ISSUE THREAD

**Issue by LaCandela** [Sense-GVT/Fast-BEV#72]
I understand that Fast-BEV uses 4 consecutive frames to improve performance. Do you correct the BEV features of the past frames before temporal fusion (concatenation)? I would naively think that some correction might be necessary to compensate for the ego-car motion between frames. Could you please guide me where this correction happens in the code?
Is it applied for every model or just for the larger ones?

**Comment by goodpeter-sun** [comment]
I don't think so since the data preparation doesn't involve can_bus info. Correct me if I am wrong, I am looking for the same point as well.

**Comment by loserhou** [comment]
> 我不要这么认为，因为数据准备不涉及can_bus信息。如果我错了，请纠正我，我也在寻找同样的观点。

Do you understand now？I don't think it's aligned, it's just concat

**Comment by goodpeter-sun** [comment]
I understand now.

**Comment by zyh321** [comment]
> > 我不要这么认为，因为数据准备不涉及can_bus信息。如果我错了，请纠正我，我也在寻找同样的观点。
> 
> Do you understand now？I don't think it's aligned, it's just concat

在进行数据准备的时候， 从雷达坐标系到图像坐标， 这个变换过程直接包括了从当前雷达点云，到上一帧的图像位置的

**Comment by venkataramanas1** [comment]
Have you up the project ? can you provide me steps to build and run properly ?
